.github/actions/e2e/action.yml

```diff
@@ -153,6 +153,15 @@ runs:
           else
             bash ./scripts/travis/e2e/e2e.sh "$E2E_TEST_ID" || exit 1          
           fi
+
+    - name: upload artifacts on gh
+      id: upload_gh      
+      uses: actions/upload-artifact@v3
+      with: 
+        name: my-artifact
+        path: /home/runner/work/alfresco-ng2-components/alfresco-ng2-components/e2e-output-*
```

Comment:
where this path is coming from? is it an error that is has duplicate `alfresco-ng2-components` inside?

Reply:
reports are created by the pipeline and uploaded at /home/runner/work/alfresco-ng2-components/alfresco-ng2-components/e2e-output-*
it is not an error